src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectGrantController.java

```diff
@@ -91,14 +99,26 @@ public CatalogObjectGrantMetadata createCatalogObjectGrantForAUser(
             }
```

Comment:
Looks like we can't comment on code that we expended in the PR.

I know it was added before but please change thrown exception `DataIntegrityViolationException` for `PublicBucketGrantAccessException` everywhere we try to add grant to a public bucket.

Reply:
Ok I'll do it.